modules/mobile/app/models/mobile/v0/adapters/appointment_requests.rb

```diff
@@ -19,12 +19,15 @@ def parse(requests)
             else
               va_appointments << Templates::VAAppointment.new(request).appointment
             end
+          rescue => e
+            Rails.logger.error(
+              'Error adapting appointment request',
+              appointment_request: request, error: e.message, backtrace: e.backtrace
+            )
+            next
           end
 
           [va_appointments, cc_appointments]
-        rescue => e
```

Comment:
This was the other problem. If we encountered any adapter issues, we rescued and logged but didn't re-raise. Which meant we returned nil instead of an array of two arrays.